client/src/modules/journal/journal.js

```diff
@@ -61,7 +61,9 @@ function JournalController(
   vm.enterprise = Session.enterprise;
   vm.languages = Languages;
   vm.gridApi = {};
+  vm.canUnpostTransactiosn = false;
```

Comment:
This is good, thank you 👍 

There is a typo in the word: `canUnpostTransactiosn` -> `canUnpostTransactions`.